src/ReactDOMTestUtils.re

```diff
@@ -2,7 +2,7 @@ type undefined = Js.undefined(unit);
 
 let undefined: undefined = Js.Undefined.empty;
 
-[@mel.module "react-dom/test-utils"]
+[@mel.module "react"]
```

Comment:
If this is not gonna be in `react-dom` anymore, I find it confusing to keep it in `ReactDOMTestUtils`. Should we move it to `React` for consistency / principle of less surprise?

Reply:
Good point, I should probably keep both and add a deprecation

Reply:
idk if deprecation is necessary. A ton of other things are breaking upstream for React 19, so maybe we can just move it directly.

Reply:
agree this should be moved to react.re if it's coming from `require('react')`

Reply:
There are a few "react" in ReactDOM module. I might probably propose to have this as a strict rule and align those as well in a future PR